File: task1/src/main/java/com/kolotree/task1/dto/Vacation/VacationShowDto.java
package com.kolotree.task1.dto.Vacation;

import com.kolotree.task1.model.VacationRequestStatus;
import lombok.AllArgsConstructor;
import lombok.Data;

import java.util.Date;

@Data
@AllArgsConstructor
public class VacationShowDto {

    private long id;

    private Date startDate;

    private Date endDate;

    private int vacationLength;

    private VacationRequestStatus vacationRequestStatus;

}

File: company-portal/src/app/modules/vacations/vacation-dto/vacation-show-dto.ts
import { UserDto } from "../../employees/employee-dto/UserDto";

    export interface VacationShowDto {
        id: number;
        user: UserDto;
        startDate: Date;
        endDate: Date;
        vacationLength: number;
        vacationRequestStatus: VacationRequestStatus;
    }
